sdm-api-doorbell.groovy

```diff
@@ -77,12 +77,12 @@ def refresh() {
 }
 
 def processChime() {
-    logDebug('Doorbell/chime pressed')
+    logDebug('${device.label}: Doorbell/chime pressed')
```

Comment:
as discussed in #32, doubling up on ${device.label} here (and all similar logDebug lines)

Reply:
My mistake, it is covered in the debug function.